rxnetty/src/main/java/io/reactivex/netty/client/ConnectionPoolImpl.java

```diff
@@ -110,54 +120,73 @@ public ConnectionPoolImpl(RxClient.ServerInfo serverInfo, PoolConfig poolConfig,
     @Override
     public Observable<ObservableConnection<I, O>> acquire() {
 
-        if (isShutdown.get()) {
+        if (isShutdownRequested.get()) {
```

Comment:
This should be moved inside the `call` method below, so that it gets checked for every subscription of the returned `Observable`.